includes/Core/Util/Debug_Data.php

```diff
@@ -347,4 +349,26 @@ function ( Module_With_Debug_Fields $module ) {
 
 		return array_merge( array(), ...$fields_by_module );
 	}
+
+
+	/**
+	 * Gets capabilities for the current user.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return array
+	 */
+	private function get_capabilities_field() {
+		$user_data = apply_filters( 'googlesitekit_user_data', array() );
```

Comment:
While this works and requires minimal changes to implement, I don't think we should couple the check here to the filter for user data.

Instead, let's extract the function in `Permissions` which is attached to that filter to a new public method that returns the map of `$permission => $current_user_can` (i.e. the result of the `array_combine` call). Then `Permissions` can be updated to call this method in its callback for the filter.

In this method, we can then simply instantiate a new `Permissions` instance and call the new method – perhaps something like `check_all_for_current_user` would be a good name. The new method should go right after the definition for `Permissions::register` since it's public. The `Debug_Data` class already has all the necessary dependencies for instantiating `Permissions` so nothing extra would be needed for that.